go.mod

```diff
@@ -1,16 +1,17 @@
 module github.com/stormeye2000/cspr-sdk-standard-tests-go
 
-go 1.20
+go 1.21.6
 
 require (
 	github.com/acarl005/stripansi v0.0.0-20180116102854-5a71ef0e047d
 	github.com/cucumber/godog v0.12.6
-	github.com/make-software/casper-go-sdk v1.3.0
+	github.com/make-software/casper-go-sdk v1.4.5-0.20240126123809-b1f5443e71d2
```

Comment:
Do you need ".20240126123809-b1f5443e71d2"?

Reply:
yes